src/main/java/com/odysseusinc/arachne/executionengine/model/descriptor/DescriptorTool.java

```diff
@@ -35,11 +35,11 @@ public static void main(String[] args) throws IOException {
                 executionRuntime = RExecutionRuntime.fromREnvLock(lock);
             } else {
                 RExecutionRuntime tempExecutionRuntime = RExecutionRuntime.fromREnvLock(lock);
-                executionRuntime.getDependencies().addAll(tempExecutionRuntime.getDependencies());
+                executionRuntime.setDependencies(tempExecutionRuntime.getDependencies());
```

Comment:
This changes logic. When multiple renv.lock files are present, only the latest dependencies will be added. However, I'm not sure that it's reasonable to expect several renv.lock.